packages/core/src/Modules/Cashier/Components/side-note.jsx

```diff
@@ -10,21 +9,32 @@ const SideNoteBullet = ({ children }) => (
     </div>
 );
 
-const SideNote = ({ notes }) => (
+const SideNote = ({ notes, title, has_bullets = true }) => (
     <div className='cashier-side-note'>
-        <DesktopWrapper>
-            <div className='cashier__header'>
-                <Localize i18n_default_text='Notes' />
-            </div>
-        </DesktopWrapper>
-        {notes.map((note, i) => (
-            <SideNoteBullet key={i}>{note}</SideNoteBullet>
-        ))}
+        {title && (
+            <DesktopWrapper>
+                <Text className='cashier-side-note__text' weight='bold' as='p'>
+                    {title}
+                </Text>
+            </DesktopWrapper>
+        )}
+
+        {notes.map((note, i) =>
+            has_bullets ? (
+                <SideNoteBullet key={i}>{note}</SideNoteBullet>
+            ) : (
+                <Text key={i} className='cashier-side-note__text' size='xxs' as='p'>
```

Comment:
Is the line height (`m`) correct here?

Reply:
thats right, line-height is m, (1.5) as per zeplin